### PR TITLE
Create website and scan storage doc types

### PR DESCRIPTION
**Pull request by pownkel**
#### Details

Adds a storage-documents package with document types to store website/page data and scan metadata

##### Motivation

Enable storage and retrieval of data required for scans

##### Context

Future related work:
- Deploy storage account
- Create data providers for each document type
- Create a REST API to retrieve documents

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`yarn precheckin`)
- [ ] Validated in an Azure resource group

### CODE REVIEW

File: packages/storage-documents/src/website.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    divisionId?: number;
+    siteStateId?: number;
+    noBanner: boolean;
+    noBannerReasonId?: number;
+    serviceTreeId: string;
+    notes?: string;
+    alias?: string;
+    organizationId?: number;
+    celaAccessibilityContacts?: string[];
+    privagyEGRCExceptionId?: string;
```
Comment (lamaks) [inline packages/storage-documents/src/website.ts:30]: typo

File: packages/storage-documents/src/website.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    AIEGRCExceptionId?: boolean;
```
Comment (lamaks) [inline packages/storage-documents/src/website.ts:31]: accessibility EGRC?

File: packages/storage-documents/src/website.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    scanners: ScanType[];
+    domainId: string;
+    owners?: string[];
+    orgCvp?: string;
+    usingUhf?: boolean;
+    compliant: boolean;
+    enableCrawler: boolean;
+    deleted: boolean;
+    divisionId?: number;
+    siteStateId?: number;
```
Comment (lamaks) [inline packages/storage-documents/src/website.ts:22]: just stateId?

File: packages/storage-documents/src/website.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    name: string;
+    baseUrl: string;
+    priority: number;
+    discoveryPatterns: string[];
+    knownPages: string[];
+    scanners: ScanType[];
+    domainId: string;
+    owners?: string[];
+    orgCvp?: string;
+    usingUhf?: boolean;
```
Comment (lamaks) [inline packages/storage-documents/src/website.ts:17]: what is Uhf?

**Review by lamaks — APPROVED** [review]

File: packages/storage-documents/src/website.ts (hunk 4 above)
Comment (udayaga) [reply packages/storage-documents/src/website.ts:17]: It is a shared component used by many sites

**Pull request merged**